apps/nextjs-website/src/components/atoms/TabComponent/TabComponent.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+import React, { FC, ReactNode } from 'react';
+import { Box, Tab, Tabs, Typography } from '@mui/material';
+
+export type TabPanelProps = {
+  readonly children?: ReactNode;
+  readonly index: number;
+  readonly value: number;
+};
+
+const TabPanel: FC<TabPanelProps> = ({ children, value, index }) => {
+  return (
+    <div
+      role='tabpanel'
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+    >
+      {value === index && <Box p={3}>{children}</Box>}
+    </div>
+  );
+};
+
+export type TabItem = {
```

Comment:
Why do you export this type?
```suggestion
type TabItem = {
```